test/e2e/policies.test-suite/topology-aware/n4c16/test12-health-checking/code.var.sh

```diff
@@ -0,0 +1,25 @@
+# Test that
+# Syntatically incorrect configuration file is resulting in the nri-resmgr
+# pod liveness probe failuire.
+host-command "$SCP \"${SRC_DIR}/test/e2e/policies.test-suite/topology-aware/n4c16/test12-health-checking/nri-resmgr-configmap.yaml\" $VM_HOSTNAME:/etc/nri-resmgr/" || {
+    command-error "copying nri-resmgr-configmap.yaml to VM failed"
+}
+
+terminate nri-resmgr
+launch nri-resmgr
+vm-command "kubectl apply -f /etc/nri-resmgr/nri-resmgr-configmap.yaml"
+
+# Give enough time for the Kubernetes to consider the container unhealthy
+sleep 30
```

Comment:
Yes, I will decrease it maybe to 20, because for the pod to fail it takes at least 15 sec because of the way I have configured the liveness probe the in deployment file.

Reply:
I think it should be possible to shorten that using short explicitly set `periodSeconds` and 
a `failureThreshold` of 1.

Reply:
yes indeed. Currently I have used 3 but I will decrease it. It was just random number I was using.

Reply:
The shorter the better, this test can have as short value as needed.